### DISCUSSION TITLE
Feature Request: Add keyboard shortcut to toggle dashboard tooltip visibility

### DISCUSSION THREAD

**Discussion by n3farious** [grafana/grafana#55124] [Dashboards, Panels & Visualizations]
**Scenario**: While performing triage and either screen sharing to highlight issues, or taking screenshots, the tooltip can obstruct parts of the data if using the mouse to draw attention to an area, or show the common cursor.

**Suggestion**: Add a keyboard shortcut of 'd t' to toggle display of tooltip while viewing dashboards.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
